File: python day 1/exercise2.py
i = 50
sum = 0

while i >= 1:
    x = str(i)
    y = input("Enter number of "+x+" euro banknotes :")
    z = i*int(y)
    if i > 20:
        i = i - 30
    else:
        i = i // 2
    sum = sum + z

print("You have", sum, "euros")

File: Java/Python1ToJava/Exercise2.java
import java.util.Scanner;
public class Exercise2{
  public static void main(String args[]){
    int i = 50;
    int sum = 0;
    while(i >= 1){
      System.out.print("Enter number of "+i+" eurobanknotes :");
      Scanner number = new Scanner(System.in);
      int euro = number.nextInt();
      int z = i * euro;
      if(i > 20){
        i = i - 30;
      }else{
        i = i / 2;
      }
      sum = sum + z;
    }
    System.out.println("You have "+sum+" euros");
  }
}
